build.gradle

```diff
@@ -112,6 +112,7 @@ allprojects {
     }
 
     tasks.withType(Javadoc) {
+        options.addStringOption('encoding', 'UTF-8')
```

Comment:
Let’s add this to our baseline java encoding plugin which does something similar for javac

Reply:
Ahh, didn't know baseline did that. I assume we can add for JavaDoc task in https://github.com/palantir/gradle-baseline/blob/0f716d5ec6063b4dae06b5f761f252178aac3941/gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineEncoding.java#L28

Reply:
Yep!

Reply:
Created https://github.com/palantir/gradle-baseline/pull/1879